coderd/templates_test.go

```diff
@@ -1618,6 +1618,69 @@ func TestPatchTemplateMeta(t *testing.T) {
 		require.NoError(t, err)
 		assert.False(t, updated.UseClassicParameterFlow, "expected false")
 	})
+
+	t.Run("SupportEmptyOrDefault", func(t *testing.T) {
+		t.Parallel()
+
+		client := coderdtest.New(t, nil)
+		user := coderdtest.CreateFirstUser(t, client)
+		version := coderdtest.CreateTemplateVersion(t, client, user.OrganizationID, nil)
+
+		description := "test-description"
+		icon := "/icon/icon.png"
+		defaultTTLMillis := 10 * time.Hour.Milliseconds()
+
+		template := coderdtest.CreateTemplate(t, client, user.OrganizationID, version.ID, func(ctr *codersdk.CreateTemplateRequest) {
```

Comment:
_The 2 hardest problems in software is naming and caching... and off by 1 errors_

`template` as a name here does not indicate this template is the expected/reference/original. Can we name it something more indicative of it's function in this test?

```suggestion
		reference := coderdtest.CreateTemplate(t, client, user.OrganizationID, version.ID, func(ctr *codersdk.CreateTemplateRequest) {
```